wallet-services/src/main/java/org/exoplatform/wallet/service/WalletContractServiceImpl.java

```diff
@@ -151,6 +159,23 @@ public ContractDetail getContractDetail(String address) {
     return null;
   }
 
+  @Override
+  public ContractDetail setDefaultValues(ContractDetail contractDetail) {
```

Comment:
Could you split it in two functions :

- setNetworkCryptoCurrency
- setCustomTokenSymbol